__tests__/searchbar.test.js

```diff
@@ -41,14 +46,43 @@ describe('Test stock search/filter', () => {
   });
 });
 
-test('Searchbar renders correctly', () => {
-  const placeholder = screen.getByPlaceholderText('Search...');
-  expect(placeholder).toBeTruthy();
+test('Displays stocks on user input/handles state chagne', async () => {
```

Comment:
I understand the benefit of wrapping your outer test function in async/await, I'm curious what the nested async/await achieves.

Reply:
The commented out axios function it what it is intended for, but breaks the tests because it isn't receiving a thenable Promise.... but it does in the next test.  So it is very much a WIP right now.  But it does currently get me an extra 2% completion on my tests